samples/cpp/benchmark_app/main.cpp

```diff
@@ -188,6 +189,22 @@ void setDeviceProperty(ov::Core& core,
     }
 }
 
+void warn_no_batch_for_dynamic(const std::vector<benchmark_app::InputsInfo>& app_inputs_info) {
```

Comment:
Why for dynamic? This warning is applicable for any model without batch

Reply:
It was under `if (isDynamicNetwork)`:
```C++
if (isDynamicNetwork) {
                    batchSize = get_batch_size(inputs);
                    if (!std::any_of(inputs.begin(),
                                     inputs.end(),
                                     [](const std::pair<const std::string, benchmark_app::InputInfo>& info) {
                                         return ov::layout::has_batch(info.second.layout);
                                     })) {
                        slog::warn
                            << "No batch dimension was found, asssuming batch to be 1. Beware: this might affect "
                               "FPS calculation."
                            << slog::endl;
                    }
                }
```

Reply:
> It was under `if (isDynamicNetwork)`:
> 
> ```c++
> if (isDynamicNetwork) {
>                     batchSize = get_batch_size(inputs);
>                     if (!std::any_of(inputs.begin(),
>                                      inputs.end(),
>                                      [](const std::pair<const std::string, benchmark_app::InputInfo>& info) {
>                                          return ov::layout::has_batch(info.second.layout);
>                                      })) {
>                         slog::warn
>                             << "No batch dimension was found, asssuming batch to be 1. Beware: this might affect "
>                                "FPS calculation."
>                             << slog::endl;
>                     }
>                 }
> ```

But for static it was printed inside get_batch_size

Reply:
Never mind, updated